cmd/main_test.go

```diff
@@ -40,6 +42,7 @@ type HTTPheader struct {
 
 var k8sClient client.Client
 var testEnv *envtest.Environment
+var ns1, ns2, ns3, ns4 k8sapi.Namespace
```

Comment:
I wonder if we should use it like that, or setup (and teardown) the namespaces for each test individually. 
@avi-biton @yftacherzog would like your opinion on that

Reply:
I suggest getting a clean environment for each test so it would not be effected by other tests.
So setup/teardown

Reply:
@avi-biton If I declare ns1, ns2, ns3, ns4 in BeforeSuite, it was not accessible in test case Entry.

Reply:
@hmariset that's why working with Before\After suite for the unit tests specifically may prove better